analytic_partner_hr_timesheet/models/res_partner.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models, api, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    timesheets = fields.One2many(
+        comodel_name='hr.analytic.timesheet', inverse_name='other_partner_id',
+        string='Timesheet activities')
+    timesheet_count = fields.Integer(
+        string='Activities', compute='compute_timesheet_count',
```

Comment:
I suggest to change this string to "Timesheet activities number", because field labels are alphabetical order in advanced search. This will help to user to understand what this label in advanced search context.